### ISSUE TITLE
Option for changing _build to .build

### ISSUE THREAD

**Issue by peabody124** [executablebooks/mystmd#527]
## Proposal

It would be great to have an option to cause Myst to use .build as the directory instead of _build 

## Additional notes

Dot as a prefix is more common in Unix for hidden directories, although one can imagine many cases where you don't want _build hidden by default. However, this would be particularly useful for things like keep myst documents organized in obsidian. It currently indexes anything without a . directory and that makes it much slower. Unfortunately their exclude files methods don't work reliably enough to do this yet.

Totally reasonable to get a "wontfix" since this is at least as much an obsidian problem.

**Comment by nthiery** [comment]
More generally, making the build directory configuration (e.g. through a command
line option) would be natural. As a use case,  I am now using both JupyterBook and
myst in parallel, and I am a bit worried to share the same _build path for both.

**Comment by fwkoch** [comment]
We may also want to make this more granular to specify different locations for different folders in the `_build` directory.

For example, I want my `site` contents in the default `_build` folder, but I want to use a shared `templates` folder across multiple projects.
